Zend/zend_language_parser.y

```diff
@@ -595,6 +595,7 @@ class_modifiers:
 class_modifier:
 		T_ABSTRACT 		{ $$ = ZEND_ACC_EXPLICIT_ABSTRACT_CLASS; }
 	|	T_FINAL 		{ $$ = ZEND_ACC_FINAL; }
+	|	T_READONLY 		{ $$ = ZEND_ACC_READONLY_CLASS|ZEND_ACC_NO_DYNAMIC_PROPERTIES; }
```

Comment:
Hm, is it intentional that `final readonly class` is not allowed?

Reply:
Hmm.. I don't understand why it wouldn't be allowed. Doesn't the `class_modifiers class_modifier` case above stand for this reason? Nevertheless, I added a test for this (it passed locally).

Reply:
Yeah sorry, I got confused over some differences between the grammar in php-src and PHP-Parser :)

Reply:
np! At least the implementation got some review ^^